src/domain/entities/product-tags.ts

```diff
@@ -73,6 +73,88 @@ export class ProductStoresTag extends BaseProductTag {}
 
+// From Issue #22
+@ProductTag('checkers_tags')
+export class ProductCheckersTag extends BaseProductTag {}
+@ProductTag('cities_tags')
+export class ProductCitiesTag extends BaseProductTag {}
+@ProductTag('correctors_tags')
+export class ProductCorrectorsTag extends BaseProductTag {}
+@ProductTag('debug_tags')
+export class ProductDebugTag extends BaseProductTag {}
+@ProductTag('informers_tags')
+export class ProductInformersTag extends BaseProductTag {}
+@ProductTag('ingredients_from_palm_oil_tags')
+export class ProductIngredientsFromPalmOilTag extends BaseProductTag {}
+@ProductTag('ingredients_that_may_be_from_palm_oil_tags')
+export class ProductIngredientsThatMayBeFromPalmOilTag extends BaseProductTag {}
+// Not found in PO
+//@ProductTag('known_nutrients')
+//export class Product Tag extends BaseProductTag {}
+@ProductTag('last_image_dates_tags')
+export class ProductLatestImageDatesTag extends BaseProductTag {}
+// This is a scaler
+//@ProductTag('numbers_of_ingredients')
+//export class Product Tag extends BaseProductTag {}
+@ProductTag('nutrient_levels_tags')
+export class ProductNutrientLevelsTag extends BaseProductTag {}
+// This is a hash
+//@ProductTag('packager_codes')
+//export class Product Tag extends BaseProductTag {}
+@ProductTag('periods_after_opening_tags')
+export class ProductPeriodsAfterOpeningTag extends BaseProductTag {}
+@ProductTag('photographers_tags')
+export class ProductPhotographersTag extends BaseProductTag {}
+@ProductTag('pnns_groups_1_tags')
+export class ProductPnnsGroups1Tag extends BaseProductTag {}
+@ProductTag('pnns_groups_2_tags')
+export class ProductPnnsGroups2Tag extends BaseProductTag {}
+@ProductTag('purchase_places_tags')
+export class ProductPurchasePlacesTag extends BaseProductTag {}
+@ProductTag('unknown_nutrients_tags')
+export class ProductUnknownNutrientsTag extends BaseProductTag {}
+// Thing this is editors
+//@ProductTag('contributors')
+//export class Product Tag extends BaseProductTag {}
+@ProductTag('popularity_tags')
+export class ProductPopularityTag extends BaseProductTag {}
+@ProductTag('ingredients_analysis_tags')
+export class ProductIngredientsAnalysisTag extends BaseProductTag {}
+@ProductTag('data_quality_bugs_tags')
+export class ProductDataQualityBugsTag extends BaseProductTag {}
+@ProductTag('data_quality_warnings_tags')
+export class ProductDataQualityWarningsTag extends BaseProductTag {}
+// Can;t see these in PO
+//@ProductTag('data_quality_warnings_producers')
+//export class Product Tag extends BaseProductTag {}
+//@ProductTag('data_quality_errors_producers')
+//export class Product Tag extends BaseProductTag {}
+//@ProductTag('possible_improvements')
+//export class Product Tag extends BaseProductTag {}
+//@ProductTag('imports')
+//export class Product Tag extends BaseProductTag {}
+@ProductTag('categories_properties_tags')
+export class ProductCategoriesProperitesTag extends BaseProductTag {}
+// This is a scalar
+//@ProductTag('owners_tags')
```

Comment:
ah that's a bug in PO, it was intended to be an array, with the value from the "owner" field

Reply:
We already have it mapped as a scalar in Product